src/Microsoft.IdentityModel.Tokens.Saml/Saml/LogMessages.cs

```diff
@@ -91,6 +91,10 @@ internal static class LogMessages
         internal const string IDX11516 = "IDX11516: We could not write the SamlStatement of type:'{0}'. You will need to override this method to write this statement.";
         internal const string IDX11517 = "IDX11517: Exception thrown while writing '{0}' for SamlSecurityToken. Inner exception: '{1}'.";
         internal const string IDX11518 = "IDX11518: Unable to writen SamlAssertion: SamlSubject.Name and SamlSubject.ConfirmationMethods.Count == 0.";
+        internal const string IDX11519 = "IDX11519: Authentication method is null when creating AuthenticationStatement for SamlSecurityToken.";
```

Comment:
You will need rebase against dev and adjust the IDX values as  https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/730 has been committed.  There is a Range defined in each LogMessage.cs file.

Reply:
@brentschmaltz rebased. These IDX values are in the range, no need to change anything.